WalletWasabi.Gui/Tabs/SettingsViewModel.cs

```diff
@@ -100,19 +100,19 @@ public SettingsViewModel(Global global) : base(global, "Settings")
 					var pinBoxText = PinBoxText?.Trim();
 					if (string.IsNullOrWhiteSpace(pinBoxText))
 					{
-						PinWarningMessage = "Please provide PIN.";
+						NotificationHelpers.Error("Please provide a PIN");
```

Comment:
```suggestion
						NotificationHelpers.Error("Please provide a PIN.");
```

Reply:
Not sure why your seeing this, that change is definately already in the code.